app/src/services/cpk.ts

```diff
@@ -1360,11 +1360,13 @@ class CPKService {
         // get mainnet relay signer
         const to = await this.cpk.ethLibAdapter.signer.signer.getAddress()
 
+        const withdrawAmount = amount.sub(RELAY_FEE)
```

Comment:
It looks like subtracting the relay fee causes problems with the minimum withdraw amount. If a user tries to withdraw the minimum 10 DAI, it's really only withdrawing 9.999, which causes the transaction to revert.

Reply:
As a note, the currently discussed solution is to only charge a relay fee if the amount to withdraw is not exactly 10 xDai, but seems like the relayer infrastructure doesn't allow omitting a fee: https://github.com/hexyls/omen-relay/blob/10429f7cbe24a52b3e77e42439dea540c5fb64c2/api/index.js#L35